server/accounts.go

```diff
@@ -2936,17 +2941,33 @@ func (s *Server) updateAccountClaimsWithRefresh(a *Account, ac *jwt.AccountClaim
 			incompleteImports = append(incompleteImports, i)
 			continue
 		}
+		from := string(i.Subject)
+		to := i.GetTo()
 		switch i.Type {
 		case jwt.Stream:
-			s.Debugf("Adding stream import %s:%q for %s:%q", acc.Name, i.Subject, a.Name, i.To)
-			if err := a.AddStreamImportWithClaim(acc, string(i.Subject), string(i.To), i); err != nil {
-				s.Debugf("Error adding stream import to account [%s]: %v", a.Name, err.Error())
-				incompleteImports = append(incompleteImports, i)
+			if i.LocalSubject != "" {
```

Comment:
Maybe use _EMPTY_ which is convention in server.

Reply:
fixed